docs/source/backends.rst

```diff
@@ -188,7 +194,7 @@ simple:
 Special (GPU) backends for numpy arrays
 =======================================
 
-A special case is if you want to supply numpy arrays and get numpy arrays back,
+A special case is if numpy arrays are required for the input and ouput,
```

Comment:
The second half of this sentence probably now needs "... but **you want to** use a different backend, such as performing **the** contraction on a GPU.".